components/editor/CryptoPrice.tsx

```
@@ -74,10 +106,14 @@ export function CryptoPrice ({ preset, onQuoteCurrencyChange, onBaseCurrencyChan
 
   function refreshPrice () {
     setLoadingState(true);
-    charmClient.getPricing(baseCurrency, quoteCurrency)
+
+    const symbol = getTokenInfo(paymentMethods, baseCurrency).tokenSymbol as CryptoCurrency;
```

please use caution when asserting types. Really I think it should only be used if it is 100% true, otherwise we're just covering over what TS is trying to protect us from. In this case,  baseCurrency is passed into the component so there's no guarantee it is valid. It's better to just throw an error in this case, since it's probably not likely but the sooner we can alert to incorrect input the easier it will be to fix

---

I've changed the signature of getPricing to a string in this [commit](https://github.com/charmverse/app.charmverse.io/pull/236/commits/5aba60edad6e50131e1cc32c1fadd8dfe0d2d260)

As we now are passing an undetermined range of crypto symbols to the pricing API, it makes more sense to use this Crypto | string type. See here for changes across the [stack](https://github.com/charmverse/app.charmverse.io/pull/236/commits/5c9c5449d7b7a1f1cea8f64b4f29005f7efdae96)